src/main/java/io/stargate/sgv2/jsonapi/config/constants/TableCommentConstants.java

```diff
@@ -0,0 +1,19 @@
+package io.stargate.sgv2.jsonapi.config.constants;
+
+public interface TableCommentConstants {
+
+  /** Top-level key for table comment json */
+  String TOP_LEVEL_KEY = "collection";
+  /** Create collection options key */
+  String OPTIONS_KEY = "options";
+  /** Collection name key */
+  String COLLECTION_NAME_KEY = "name";
+  /** Collection indexing key */
+  String COLLECTION_INDEXING_KEY = "indexing";
+  /** Collection vector key */
+  String COLLECTION_VECTOR_KEY = "vector";
+  /** Schema version key */
+  String SCHEMA_VERSION_KEY = "schema_version";
+  /** Schema version value */
+  int SCHEMA_VERSION_VALUE = 1;
```

Comment:
It seems there is nowhere to handle if the `SCHEMA_VERSION_VALUE` becomes 2,3... in the future

Reply:
Yes, currently, it is not that useful.
It is more like to plan ahead, and will be useful when we have design changes in the future.